examples/basic/tidb-cluster.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     baseImage: pingcap/tikv
     replicas: 3
     requests:
-      storage: "50Gi"
+      storage: "1Gi"
```

Comment:
as a basic hello world example, we shouldn't configure high resource requests. otherwise, it cannot run in many situations, e.g. minikube vm or kind cluster on a small machine.

Reply:
agreed.